### ISSUE TITLE
automatic 3D scenes from 2D artworks: iterative "hypothesis testing" in 3D software, and render informs loss function

### ISSUE THREAD

**Issue by dreamstimlab** [neuro-dream/IDEAS#43]
but:
- how validate different angles of the 3D view?
    - needs some kind of parsimonious solution 
    - this also should draw from model continuously pre-trained across images: sparse solution should hold across image space
